### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by daniel-bush**
@cvtorrisi93

Please check formatting, naming and logic

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
 """hex_colours exercise."""
 
-HEX_COLOURS_DICT = {"aquamarine": "#7fffd4", "beige": "#f5f5dc"}
+HEX_COLOURS_DICT = {"aquamarine": "#7fffd4", "beige": "#f5f5dc", "brown": "#a52a2a",
```
Comment (cvtorrisi93) [inline prac_05/hex_colours.py]: In one of Linday's lectures, he mentioned not to call a variable/constant name by what it is. For example, list_of_places. You could call to something that relates the key to the value, i.e. COLOUR_TO_HEX   {colour: hex}

File: prac_05/state_names.py [hunk 2]
```diff
@@ -5,14 +5,18 @@
...
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
     else:
         print("Invalid short state")
-    state_code = input("Enter short state: ")
+    state = input("Enter short state: ").upper()
+
+# loop with formatting
+for state in STATE_NAMES:
+    print("{:3} is {}".format(state, STATE_NAMES[state]))
```
Comment (cvtorrisi93) [inline prac_05/state_names.py:22]: You could use list comprehension to get the max length of the keys in the dictionary to then apply it to the .format() function for the short state code

File: prac_05/hex_colours.py (hunk 1 above)
Comment (daniel-bush) [reply prac_05/hex_colours.py]: wonderful - done!

File: prac_05/state_names.py (hunk 2 above)
Comment (daniel-bush) [reply prac_05/state_names.py:22]: great thanks,

**Pull request merged**